packages/lwc-language-server/scripts/convert.js

```diff
@@ -50,26 +50,71 @@ const tags = Object.keys(data).map(key => {
     };
 });
 
+//make globalAttribute changes here, not in standard-lwc.json as they'll be overwritten
 const globalAttributes = [
```

Comment:
nit: so what way we could have caught his is if 1. this was a ts file and 2. we defined a type/interface for the attributes to implement.

Reply:
this is more just an observation.  no action to be taken here.

Reply:
💯